### ISSUE TITLE
MUERR_RENORM crazy values

### ISSUE THREAD

**Issue by maria-vincenzi** [RickKessler/SNANA#1009]
MUERR_RENORM/MUERR crazy values for some realizations
See example of crazy values here: `/scratch/midway2/rkessler/PIPPIN_OUTPUT/D5YR_SMP/6_BIASCOR/TESTw_P21_4D_SIGINT/output_ITER1/OUTPUTBBCFIT-000`1
See example of normal values here: 
`/scratch/midway2/rkessler/PIPPIN_OUTPUT/D5YR_SMP/6_BIASCOR/TESTw_P21_4D_SIGINT/output_ITER1/OUTPUTBBCFIT-0023`
Log files are untarred here: `/scratch/midway2/rkessler/PIPPIN_OUTPUT/D5YR_SMP/6_BIASCOR/TESTw_P21_4D_SIGINT/output_ITER1/SCRIPTS_BBCFIT`

**Comment by RickKessler** [comment]
With MUCOVADD option we expect final sigInt near zero, so I think the problem is that initial sigint=0.11 (sig1 or simB input key) is too large and messes up something in the initialization. I ran a test varying initial sigmB and checking MUDIFERR for one z-bin:

 start      
 sigmb     MUDIF    MUDIFERR(zbin=10)
 0.01      0.0047   0.0147
 0.02      0.0047   0.0146  
 0.04      0.0044   0.0146  
 0.06      0.0042   0.0146  
 0.08      0.0042   0.0146  
 0.10      0.0042   0.0146  
 0.12      0.0044   0.0210 <=== crazy M0 error
 0.14      0.0044   0.0220 <=== crazy M0 error

Note the abrupt  MUDIFERR change at sigmB=0.12.  The previously used value sigmB=0.11 is on the border of correct and inflated errors, and may explain the two bands of MUDIFERR vs. sqrt(NFIT). To avoid using too-large sigmB, I set an abort trap
   if ( MUCOVADD option and sigmB > 0.05 ) --> abort
